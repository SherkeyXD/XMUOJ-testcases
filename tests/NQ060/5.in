1066354363 470
0 347202 443190 1050111 1259280 1471136 1563428 1568833 1607673 2489600 2862328 2974264 3067776 3078030 3125654 3860080 4582884 5255904 5333291 5913990 5938060 7235255 9001413 9830984 10432560 10700520 12870230 12973950 13409600 13958028 14091136 14538510 14679789 15097488 15733244 15835300 17038824 17364340 17828910 18714210 19505600 19790925 21932496 23243376 23525448 24636366 25766692 26758610 27150785 27447368 27514088 28365129 29620413 31262918 31404560 33907852 34768215 35543376 36187560 36280522 36361010 36465629 38967300 39496233 39827772 41185776 41928192 43993588 44164690 45290400 45406800 45976770 46398333 46590683 47242500 47302024 47853045 48482616 48774544 49082870 49239569 49788844 50746167 50867052 51203670 53586600 55240450 55288312 56491228 58361394 59333904 59779075 60553824 60645823 60947740 61091662 61613616 62170954 64068570 64228125 64707117 64979280 65219046 65724720 66738252 67602608 68417061 68564352 69919844 70624755 71084176 71449330 71801856 72642811 73014340 74607390 74613705 75434184 75735724 76434862 78229746 78981048 79783598 80044048 83212230 83484940 83787956 85080402 85798944 86090782 87831546 89099955 89388432 92547100 92689025 93087890 93254014 94524760 95670392 97240676 99280540 100175602 102895104 103930615 105697076 108030150 108042511 109500296 109746385 110260602 114032132 117036308 117775728 119076980 120726412 122632084 125464220 128577287 130353160 130458340 131799200 132269529 132643680 133020290 133558100 133602996 134070045 134304345 135036558 138025680 139051161 139558240 141271754 141427134 141518610 142436052 144400733 145769814 148225280 148265670 148316696 149766618 150144978 151618935 154551320 156079805 157711548 158536275 160057600 160959704 163221723 163819556 165029913 166939539 167243700 168213318 170647932 173660805 176457790 178090360 179255398 180577200 181840677 184435915 185211348 186943584 187429662 187662528 188440096 190514984 191331686 193142418 193575910 194853282 196363440 196446920 196845321 197357027 198126855 198556056 199400501 199665904 200405160 200639184 205659805 207788214 208242353 209443365 209997050 213775907 214180788 214878900 215443284 216667220 217094240 221201386 227167328 227910144 228070070 229646000 234971722 235318668 235708400 241554964 242685828 243837174 245887548 246898875 250740490 253370904 258676352 261857079 263188080 264747375 266879396 267052500 267214618 267423198 268701525 269514297 271333062 273956326 274790637 275836110 277843709 278309628 278631754 279734346 281347110 287305370 288734551 290683080 293347730 299546834 301160640 302432102 302610188 306360104 307604077 309065028 312577650 313004584 315527250 315682575 316014398 316119638 317917425 319139484 319443190 319816728 319998348 321061876 322795680 322932580 323105302 325020304 326096280 326153978 328848218 329196360 332685068 332760414 332855415 334564857 334588720 335906030 336064261 337766633 339146289 339676116 341226516 342319480 342342174 345504243 345675850 346911567 348242131 349644051 353774040 356382591 359798442 362151404 363057200 364652600 365693625 368227744 369055695 369702960 371091738 376165960 378781500 381108900 386848112 388443000 388850210 391332810 391443540 391478796 392356380 394736911 395872994 401347760 401535450 401583468 413914416 414597120 415785516 418031955 418689810 419472380 419665026 420343656 421566048 421688531 421847568 424054400 425386738 430551044 431439213 432310800 432481857 433530496 435548667 435571749 437436016 439284972 439394448 444150789 449210552 451857788 451864000 454325000 454329790 457349373 460647810 461546174 464766815 467964420 469461117 470390151 472803570 473240650 479011995 480457774 482377880 483356960 483991080 489229872 491476994 497396393 511805608 513768266 523262492 528376632 529138505 537261855 542654013 543265032 551852049 553727559 556123726 557695164 561451520 561812781 566620182 569182380 573273396 574378100 575911685 577052124 579294596 584074599 585326599 585985680 588607579 593563620 593973030 595809514 602917994 604594448 606411366 608716416 615624565 616017765 622399897 634475950 641538156 642231860 644077352 649475580 652643073 657174265 658894752 665637918 666748655 666778022 679515586 679950483 682396470 682577652 683486208 694335160 696854328 723308660 737314972 737840832 740336262 750884477 757436295 774368750 776788376 784092848 787297322 801887715 806448687 829478559 832581480 839692128 841451119 854779326 870870322 887591866 934947910 960835868 978408690 979229198 988484476 990848000 1003300776 1035599175 