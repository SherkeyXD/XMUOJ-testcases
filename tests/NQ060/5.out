2862328 173660805 432481857 457349373
43993588 74613705 353774040 593973030
79783598 156079805 376165960 454325000
93254014 138025680 281347110 553727559
